server/plugins/index.js

```diff
@@ -18,12 +18,16 @@ function checkModule (module) {
 
 function registerPlugins (server, config, callback) {
 
-  const options = Object.assign({ config }, config.pluginOptions)
+  // TODO: port away from Objec.assign (or require Node 6.0.0+ for Hoodie)
+  var options = Object.assign({ config: config }, config.pluginOptions)
 
   var hapiPlugins = [
     require('inert')
   ]
-
+  config.plugins = config.plugins || [] // doesn’t work, config.plugins is {} here
+  if (Object.keys(config.plugins).length === 0) {
+    config.plugins = [] // workaround for above issue
+  }
```

Comment:
I think that `config.plugins` should always be set, by setting it to `[]` in https://github.com/hoodiehq/hoodie/blob/master/cli/hoodie-defaults.js